chain/x/zoracle/internal/types/util.go

```diff
@@ -14,3 +14,13 @@ func uint64ToBytes(num uint64) []byte {
 	}
 	return buf.Bytes()
 }
+
+func int64ToBytes(num int64) []byte {
+	buf := new(bytes.Buffer)
+	err := binary.Write(buf, binary.BigEndian, num)
+	// Encode int64 must not have error
```

Comment:
Another way to do it, according to how Cosmos-SDK does this, is:
```
        result := make([]byte, 8)
        binary.BigEndian.PutUint64(result, uint64(num))
        return result
```

That way, we don't need to import `bytes` and don't need to deal with error.